### ISSUE TITLE
Creation of Github Teams

### ISSUE THREAD

**Issue by ericspod** [Project-MONAI/MONAI#7544]
My idea was to create new teams within the Project MONAI Github organisation. We have a few here https://github.com/orgs/Project-MONAI/teams but I think we should add more for specific areas of interest, topics, application areas, etc. so that there's a known group of people who might be able to contribute to issues or discussion. 

For example, I'd suggest a *UNet* or *Networks* team composed of people who have contributed to networks in some way or are otherwise known as experts on these topics. If a question relating to networks comes up then this team can be referred to @-referencing so that the issue is more visible to someone in that team who can help out. It would be expected that people would be members of multiple teams, but also that they wouldn't be obligated to answer every summons. I see this as an easy way of knowing who to ask for help on certain topics when they arise. 

Some team ideas:
* Networks
* Loss Functions
* Dataset and DataLoader
* Workflows (Ignite)
* Segmentation
* Classification
* Diffusion
* Generative
* Label
* Other working group topics

This is a proposal for discussion on this so please do comment. We should also be inviting more contributors for these teams as well as core contributors.

@Nic-Ma @KumoLiu @aylward

**Comment by aylward** [comment]
I like the idea - the way I think about it is not in terms of "teams" but instead in terms of "topics."   People can subscribe to a topic, and we can then tag a discussion with a topic.   These teams would be experts as well as anyone interested in the topic.   It would thereby also help more people become experts in topics that interest them. 

Is that consistent with your thoughts / goals with establishing these teams?

**Comment by ericspod** [comment]
After discussion with Nic I think we should have the following as teams added to https://github.com/orgs/Project-MONAI/teams:

* Each working group should have its own team with members of that group. This would allow reviewers and commenters know who to contact regarding features those working groups are responsible for or worked on. We have a deploy working group team already so from that list I know who would possibly help with a deployment question if it came up outside of the appropriate repos. 
* Teams should be created for areas of interest as I've described above, eg. a network team for specialists in network design, a transform team for those working on transforms closely
* A core developers team? This is different from the Core Reviewers team that's currently there which is used to reference the default set of reviewers for features.

**Comment by aylward** [comment]
This seems very reasonable.  Perhaps also come up with a process/rule for removing old/unused teams.

Not certain what side-effects this might have on Github, so perhaps we may need to rename defunct groups to "deprecated-<groupname>" or something like that.
